examples/content/src/content/jcr_root/content/_cq_tags/core-components-examples/component-type/.content.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description=""
+    jcr:primaryType="cq:Tag"
+    jcr:title="Component Type"
+    sling:resourceType="cq/tagging/components/tag">
+    <atom/>
```

Comment:
Added component type tags so that:
* We can build a list from tags
* Potentially reference these in the home screen or on the component pages as metadata